html/components/masthead.js

```diff
@@ -1,11 +1,12 @@
 /* global document window */
-import { throttle } from 'lodash';
+import { throttle, uniqueId } from 'lodash';
 import getElements from '../utilities/getElements';
 import { focusFirstEl } from '../utilities/elementState';
 import { isEscPressed } from '../utilities/keypress';
 import isElementVisible from '../utilities/isElementVisible';
 import scrollYDirection from '../utilities/scrollYDirection';
 import { hideDropDown, showDropDown } from './dropdown';
+import { toggleAriaExpanded } from './toggle';
```

Comment:
Hey Bob, we don’t want to create a dependency on toggle here. Can we make a util for toggleAriaExpanded ?

Reply:
Let's talk about this more. I agree it should be a utility, but it's exported in spark-exports already.

Reply:
@bobwise, if someone wants to use that utility they shouldn't have to import the toggle file. Even though toggle may be the only place it is used in Spark right now, it's important to remember that we don't code for the past, but for the future. And we have a lot of new components on the horizon. We are also currently auditing everything we have in Spark from an a11y standpoint and who knows what will come up! Whats the hesitation from adding this as a utility?

Reply:
talked in person. We're going to pull this functionality out into an html utility file that can be shared between the HTML toggle components. Eventually we should deprecate `toggleAriaExpanded` in toggle.js and export the new utility function in spark-exports.

Reply:
Created the `toggleAriaExpanded` and `generateAriaControls` utilities. `toggleAriaExpanded` is exported as `toggleAriaExpandedAttribute` because we already have a utility with that name.